styles.css

```diff
@@ -0,0 +1,339 @@
+  }
+  .main-container{
+   margin: 3px !important;
+   grid-gap:5px !important;
+  }
+  section{
+    display:flex;
+    flex-wrap: wrap !important;
+    width:100% !important;
+  }
+  .section-left-side{
+    flex: 0% !important;
+  }
+  .section-left-side img{
+    height: 100px !important;
+  }
+  .section-right-side{
+    flex: 50% !important;
+    flex-wrap: wrap !important;
+    padding: 0px !important;
+  }
+  .section-right-side h3{
+    font-size: medium;
+  }
+  p{
+    padding: 0px !important;
+    font-size: small !important;
+    word-wrap: break-word !important;
+  }
+  article{
+    grid-template-columns: auto auto !important;
+    grid-gap:2px !important;
+    width: 100% !important;
+  }
+  .row-1{
+    grid-template-columns: auto auto !important;
+    grid-gap:2px !important;
+    display: flex !important;
+    flex-wrap: wrap !important;
+    width: 140% !important;
+  }
+  .col{
+    width: 50%;
+  }
+  .card-desc{
+    padding:2px !important;
+  }
+  .card-desc p{
+    padding: 0px !important;
+    font-size: small !important;
+  }
+  footer{
+    width:100% !important;
+  }
+}
+/* Tablet Styles */
+@media only screen and (min-width:501px) and (max-width:959px){
+  header{
+    width:100%;
+  }
+  h4{
+    padding: 2px !important;
+  }
+  nav{
+    width: 100% !important;
+    padding: 5px;
+  }
+  body{
+    width: 100%;
+    background: lightpink;
+  }
+  .main-container{
+    margin: 3px !important;
+    grid-gap:5px !important;
+  }
+  section{
+    flex-wrap: wrap !important;
+    width:100% !important;
+  }
+  .section-left-side{
+    flex: 0% !important;
+  }
+  .section-left-side img{
+    height: 145px !important;
+  }
+  .section-right-side{
+    flex: 50% !important;
+    flex-wrap: wrap !important;
+    padding: 0px !important;
+  }
+  .section-right-side h3{
+    font-size: medium;
+  }
+  p{
+    padding: 0px !important;
+    font-size: small !important;
+    word-wrap: break-word !important;
+  }
+  article{
+    grid-gap:5px !important;
+    width: 100% !important;
+  }
+  .row-1{
+    grid-gap:5px !important;
+  }
+  .col{
+    width: 100%;
+  }
+  .card-desc{
+    padding:2px !important;
+  }
+  .card-desc p{
+    padding: 0px !important;
+    font-size: small !important;
+  }
+  .footer{
+    width:100%;
+  }
+}
+/* Desktop Styles */
+@media only screen and (min-width: 960px) {
+  body{
+    width:100%;
+  }
+  header{
+    width:100%;
+  }
+  .main-container{
+    width:70%;
+  }
+  footer{
+    width:100%;
+  }
+  .banner-left-side{
+    width:50%;
+  }
+  .banner-right-side{
+    width:50%;
+  }
+}
+.main-container{
+  display: grid;
+  grid-template-rows: auto auto auto;
+  grid-gap: 30px;
+  margin : 20px 150px;
+}
+h4,h5{
+  margin:0px;
+}
+h3{
+  margin:3px;
+}
+p{
+  margin:0;
+  font-weight: 100;
+  font-size: small;
+}
+a{
+  color:white;
+  text-decoration: none;
+}
+header{
+  display: flex;
+  background-image: url('images/header-image.jpg');
+  height: 250px;
+  padding:40px;
```

Comment:
Adding padding to your header is throwing off the the dimensions for the rest of the page. Try disabling it and your navbar should now reach across the entire width.